packages/agoric-cli/src/install.js

```diff
@@ -40,7 +40,112 @@ export default async function installMain(progname, rawArgs, powers, opts) {
   }
 
   let subdirs;
+  const installDirs = ['.'];
   let sdkWorktree;
+  const versions = new Map();
+  const linkFolder = path.resolve(`_agstate/yarn-links`);
+  const linkFlags = [];
+  if (opts.sdk) {
+    linkFlags.push(`--link-folder=${linkFolder}`);
+  }
+
+  const doYarnInstalls = async (phase, ...flags) => {
+    for (const instdir of installDirs) {
+      log.info(`yarn install ${phase} in ${instdir}`);
+      // eslint-disable-next-line no-await-in-loop
+      const yarnInstall = await pspawn(
```

Comment:
not worth doing concurrently?

Reply:
I originally made it concurrent, but that just made the output extremely difficult to read.